File: Emlak/src/app/admin/model/IResim.ts
export interface IResim {
    ID: number,
    //[Required(ErrorMessage = "Baþlýk alaný boþ olamaz ve en fazla 50 karakter olmalýdýr.")]
    //[StringLength(50)]
    Title: string,
    //[DataType(DataType.MultilineText)]
    //[AllowHtml]
    Description: string,
    PictureUrl: string,
    ThumbUrl: string,
    Code: string,
    Active: boolean,
    Deleted: boolean,

    Mesaj: string,

    OldPictureUrl: string,
    OldThumbUrl: string,
    HasFile: boolean,

    TypesList: string[]
}

File: AdminPanel/Repository/Resim/IResim.cs
using System.Collections.Generic;
using System.Web.Mvc;

namespace Repository.ResimModel
{
    public interface IResim
	{
		 int ID { get; set; }
		 string Title { get; set; }
		 string Description { get; set; }
		 string PictureUrl { get; set; }
		 string ThumbUrl { get; set; }
		 string Code { get; set; }
		 bool Active { get; set; }
         bool Deleted { get; set; }

         string Mesaj { get; set; }

         List<SelectListItem> TypesList { get; set; }

        List<Resim> List();
        IResim Select(int id);
        bool Insert(IResim resim);
        bool Update(IResim resim);
        bool Delete(int id);
        bool Remove(int id);
    }
}
